.travis.yml

```diff
@@ -1,11 +1,11 @@
-language: c
+language: python
```

Comment:
Yes, it's a pain...
Looks like this is a long standing issue (they discuss some workarounds):
https://github.com/travis-ci/travis-ci/issues/2312